File: src/main/webapp/formulier/formulier.js
function addFormulier() {
    $.get("localhost:8888/standenmotor/addFormulier", function (data) {
        var formulierTable = $("#formulierTable");
        $.each(data, function (index, object) {
            var String =
                "<tr id='row" +
                index +
                "'>" +
                "<td class='code_row'>" +
                object.formulierid +
                "</td>" +
                "<td>" +
                object.wedstrijdid +
                "</td>" +
                "<td>" +
                object.opmerkingid +
                "</td>" +
                "<td>" +
                object.teamidthuis +
                "</td>" +
                "<td>" +
                object.teamiduit +
                "</td>" +
                "<td>" +
                object.doelpuntenthuis +
                "</td>" +
                "<td>" +
                object.doelpuntenuit +
                "</td>" +
                "<td class='delete'>&times;</td>" +
                "</tr>";
            formulierTable.append(String);
        });
    });
}

File: src/main/java/ipass/standenmotor/webservice/Formulier.java
package ipass.standenmotor.webservice;

public class Formulier {

	public int formulierid;
	public int wedstrijdid;
	public int opmerkingid;
	public int teamidthuis;
	public int teamiduit;
	public int doelpuntenthuis;
	public int doelpuntenuit;
	
	public Formulier (int fid, int wid, int oid, int tidt, int tidu, int dpt , int dpu){ 
	formulierid = fid;
	wedstrijdid = wid;
	opmerkingid = oid;
	teamidthuis = tidt;
	teamiduit = tidu;
	doelpuntenthuis = dpt;
	doelpuntenuit = dpu;
}

	public int getFormulierID() {
		return formulierid;
	}
	public int getWedstrijdID() {
		return wedstrijdid;
	}
	public int getOpmerkingID() {
		return opmerkingid;
	}
	public int getTeamIDThuis() {
		return teamidthuis;
	}
	public int getTeamIDUit() {
		return teamiduit;
	}
	public int getDoelpuntenThuis() {
		return doelpuntenthuis;
	}
	public int getDoelpuntenUit() {
		return doelpuntenuit;
	}
}